app/src/main/java/io/apicurio/registry/rules/validity/KafkaConnectContentValidator.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2020 Red Hat
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.apicurio.registry.rules.validity;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import javax.enterprise.context.ApplicationScoped;
+
+import org.apache.kafka.connect.json.JsonConverter;
+import org.apache.kafka.connect.json.JsonConverterConfig;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import io.apicurio.registry.content.ContentHandle;
+
+/**
+ * A content validator implementation for the Kafka Connect schema content type.
+ * @author eric.wittmann@gmail.com
+ */
+@ApplicationScoped
+public class KafkaConnectContentValidator implements ContentValidator {
+    
+    private static final ObjectMapper mapper = new ObjectMapper();
+    private static final JsonConverter jsonConverter = new JsonConverter();
+    static {
+        Map<String, Object> configs = new HashMap<>();
+        configs.put("converter.type", "key");
+        configs.put(JsonConverterConfig.SCHEMAS_CACHE_SIZE_CONFIG, new Integer(0));
+        jsonConverter.configure(configs);
```

Comment:
Hmmm ... does Java really always create static fields before running static init block?
So we don't end-up with NPE on jsonConverter at line 45 ... on some random occasion ...

Reply:
They are executed in textual order.